docs/api/post-v1-querybatch.md

```diff
@@ -157,4 +159,61 @@ EOF
         }
     ]
 }
+```
+
+## Pagination
+
+Pagination for the querybatch API works similarly to the individual query, with the distinction that not all queries in the querybatch are likely to return a `next_page_token`. 
+
+A queryset response with paginated results will be in this form:
+
+```json
+{
+  "results": [
+    {
+      "vulns": [
+        ...
+      ],
+      "next_page_token": "token for query 1"
+    },
+    {
+      "vulns": [
+        ...
+      ],
+      "next_page_token": "token for query 2"
+    },
+    {
+      "vulns": [
+        ...
+      ],
+    },
+    ...
+  ]
+}
+```
+Notice that each result has a distinct "next_page_token" and that the third result does not include a `next_page_token`. This is because all of the vulnerabilities for the third query have been returned. 
+
+To get the next page of results, your next request should be in the form:
```

Comment:
Can you specify in this sentence what differs? `To get the next page of results, your next request should specify `page_token` only for the queries that returned `next_page_token`.`